activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -521,7 +521,10 @@ def drop_table(table_name, options = {})
       # * <tt>:precision</tt> -
       #   Specifies the precision for the <tt>:decimal</tt> and <tt>:numeric</tt> columns.
       # * <tt>:scale</tt> -
-      #   Specifies the scale for the <tt>:decimal</tt> and <tt>:numeric</tt> columns.
+      #   Specifies the scale for the <tt>:decimal</tt> and <tt>:numeric</tt> columns. 
```

Comment:
Can you remove the trailing whitespace that you added?

Reply:
Done. I was staring at that for like 10 minutes trying to figure out what the diff was.